website/src/components/SideMenu.tsx

```diff
@@ -21,8 +21,9 @@ export function SideMenu(props: SideMenuProps) {
   return (
     <main className="sticky top-0 sm:h-full">
       <Card
+        display={{ base: "grid", sm: "flex" }}
         width={["100%", "100%", "100px", "280px"]}
-        className="grid grid-cols-4 gap-2 sm:flex sm:flex-col sm:justify-between p-4 h-full"
+        className="grid-cols-4 gap-2 sm:flex-col sm:justify-between p-4 h-full"
```

Comment:
How is the new behvaior different than the one before? it seems to me that they are identical?

Reply:
The Card component has display flex by default and It will override tailwindcss.
Here is how it looks on the dev instance.
![image](https://user-images.githubusercontent.com/33456881/216283727-bac6d80e-2b25-423a-b30b-5cdcb0ab38db.png)